regcore/migrations/0002_auto_20160308_2228.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import mptt.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('regcore', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='regulation',
+            name='children',
+        ),
```

Comment:
You can continue to use the `children` field directly because Django's [data migrations](https://docs.djangoproject.com/en/1.9/topics/migrations/#data-migrations) effectively _serialize_ the schema. That said, we're using a custom field; it'd probably make sense to copy-paste the custom json field logic into a migration to limit dependencies. I think that'd mean we'd want to fetch `children` as a string.

Maybe 3 migrations:
1. Delete non-root models; add new columns, but don't set the "relation" field on the `parent` foreign key
2. Data migration which access the model's `children` field as a string and deserializes w/ copy-pasted method
3. Drop the `children` field, modify the `parent` field to add the "relation" of `children`

What do you think?